tests/test_service.py

```diff
@@ -104,6 +104,15 @@ def test_list_shop_cart_entry_by_user_id(self):
         #self.assertEqual(data['user_id'], 1)
         self.assertTrue(len(resp.data) > 0)
 
+     def test_delete_a_product(self):
+        """ Delete a Product """
+        product = new_product(user_id=1, product_id=1)
+        product.save()
+        self.assertEqual(len(new_product.all()), 1)
+        # delete the pet and make sure it isn't in the database
+        pet.delete()
```

Comment:
is the pet correct?

Reply:
Thanks for pointing out.I will correct it